pkg/blob/nodeserver.go

```diff
@@ -348,7 +353,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	var output string
 	if d.enableBlobfuseProxy {
-		output, err = d.mountBlobfuseWithProxy(args, authEnv)
+		output, err = d.mountBlobfuseWithProxy(args, protocol, authEnv)
 	} else {
 		output, err = d.mountBlobfuseInsideDriver(args, authEnv)
```

Comment:
what if user is not using blobfuse-proxy, will `protocol: fuse2` would do blobfuse2 mount?

Reply:
Actually not, should we also support the scenario that not using blobfuse-proxy?